src/meshes/structured_mesh.jl

```diff
@@ -23,7 +23,8 @@ mutable struct StructuredMesh{NDIMS, RealT <: Real} <: AbstractMesh{NDIMS}
 end
 
 """
-    StructuredMesh(cells_per_dimension, mapping; RealT=Float64, unsaved_changes=true, mapping_as_string=mapping2string(mapping, length(cells_per_dimension)))
+    StructuredMesh(cells_per_dimension, mapping; RealT=Float64, unsaved_changes=true, 
+                   mapping_as_string=mapping2string(mapping, length(cells_per_dimension), RealT=Float64))
```

Comment:
```suggestion
    StructuredMesh(cells_per_dimension, mapping; RealT=Float64, unsaved_changes=true, 
                   mapping_as_string=mapping2string(mapping, length(cells_per_dimension)))
```

It was there twice

Reply:
Should both `RealT=Float64` be removed or only `=Float64` ? I thought it makes sense to be explicit in the docstring as the default argument to `mapping2string` is `RealT`.

Reply:
Sorry, my mistake - too many parentheses

Reply:
```suggestion
    StructuredMesh(cells_per_dimension, mapping;
                   RealT=Float64,
                   unsaved_changes=true, 
                   mapping_as_string=mapping2string(mapping, length(cells_per_dimension), RealT=RealT))
```
This is what is implemented, isn't it?

Reply:
They are different and the original version will cause type instability
```julia
julia> function foo(; RealT=Float64)
       print("foo ", RealT)
       end
foo (generic function with 2 methods)

julia> function foo1(;RealT=Float64, f=foo(RealT=Float64))
       print("foo1 ", RealT)
       end
foo1

julia> foo1(RealT=Float32)
foo Float64foo1 Float32
```

Reply:
```julia
julia> function foo(; RealT=Float64)
       print("foo ", RealT)
       end
foo (generic function with 2 methods)

julia> function foo1(;RealT=Float64, f=foo(RealT=RealT))
       print("foo1 ", RealT)
       end
foo1

julia> foo1(RealT=Float32)
foo Float32foo1 Float32
```

Reply:
@ranocha